### PR TITLE
adding German notebooks

### PR DESCRIPTION
**Pull request by vinhngx**
Replacing https://github.com/nvidia-riva/tutorials/pull/65.
- Addressed @LynseyFabel's previous comments
- Running through the example with new container Nemo 22.08
- Update deployment to using Riva 2.6.0

### CODE REVIEW

**Comment by AshishSardana** [comment]
1. provide the details of the env/container to run `data_preparation` nbs in.
2. add `rm -rf` cmd in the 3 "download-and-preparation" nbs to delete the large raw datasets.

**Comment by AshishSardana** [comment]
`riva-deploy` is not able to optimise (tensorrt) our trained German acoustic model (`de-asr.rmir`). the error it throws is:
```
[12/14/2022-07:08:33] [TRT] [E] 10: [optimizer.cpp::computeCosts::2011] Error Code 10: Internal Error (Could not find any implementation for node shuffle_after_(Unnamed Layer* 18) [Convolution]_output.)
[12/14/2022-07:08:33] [TRT] [E] 2: [builder.cpp::buildSerializedNetwork::609] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed. )
```

it is a known error.

File: New-language-adaptation/German/data_preparation/1.1-download-and-preprocess-MCV.ipynb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ec33534",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!sudo apt-get install -y wget sox libsox-fmt-mp3 parallel"
```
[the remark this answers is not in the document]
Comment (vinhngx) [reply New-language-adaptation/German/data_preparation/1.1-download-and-preprocess-MCV.ipynb:26]: added apt-get update

File: New-language-adaptation/German/data_preparation/1.1-download-and-preprocess-MCV.ipynb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+   "id": "2ae0dd16",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir -p data/processed/mcv\n",
+    "\n",
+    "OUT_DIR = os.path.join(CUR_DIR, \"data/processed/mcv\")\n",
+    "DATA_ROOT = os.path.join(CUR_DIR, \"data/raw/mcv\")\n",
+    "\n",
+    "!python3 ./data_ingestion/process_mcv.py --data_root=$DATA_ROOT --data_temp=/tmp --data_out=$OUT_DIR --manifest_dir=$OUT_DIR --save_meta true"
```
[the remark this answers is not in the document]
Comment (vinhngx) [reply New-language-adaptation/German/data_preparation/1.1-download-and-preprocess-MCV.ipynb:90]: good observation. I've added a few notes re. data version differences

File: New-language-adaptation/German/data_preparation/3.1-filter-mix-traintest-split.ipynb [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+      "Processing  voxpopuli dev\n",
+      "Processing  voxpopuli test\n"
+     ]
+    }
+   ],
+   "source": [
+    "for subset in ['train', 'dev', 'test']:\n",
+    "    merged_manifest = []\n",
+    "    #for dataset in ['mls', 'voxpopuli', 'mcv']:    \n",
+    "    for dataset in ['voxpopuli']:\n",
```
[the remark this answers is not in the document]
Comment (vinhngx) [reply New-language-adaptation/German/data_preparation/3.1-filter-mix-traintest-split.ipynb:185]: modified the notebooks to process all datasets/subsets now.

File: New-language-adaptation/German/data_preparation/3.1-filter-mix-traintest-split.ipynb [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "#for dataset in ['voxpopuli', 'mls', 'mcv']:\n",
+    "for dataset in ['voxpopuli']:\n",
```
[the remark this answers is not in the document]
Comment (vinhngx) [reply New-language-adaptation/German/data_preparation/3.1-filter-mix-traintest-split.ipynb:146]: modified the notebooks to process all datasets/subsets now.

File: New-language-adaptation/German/deployment/deployment.ipynb [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+   "id": "c7890fff-3886-4a6e-b8f4-1946c5bea05f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile nemo_conversion.sh\n",
+    "cd /riva_quickstart\n",
+    "pip3 install nvidia-pyindex\n",
+    "pip3 install nemo2riva-2.6.0-py3-none-any.whl\n",
+    "\n",
+    "#Converting acoustic model to Nemo format.\n",
```
[the remark this answers is not in the document]
Comment (vinhngx) [reply New-language-adaptation/German/deployment/deployment.ipynb:226]: fixed

**Comment by vinhngx** [comment]
fixed most comments from @AshishSardana . Thanks Ashish!

@svenchilton  can you pls approve and merge this MR? Thanks.

**Review by svenchilton — APPROVED** [review]
LGTM

**Pull request merged**